pkg/util/helperfunctions.go

```diff
@@ -71,3 +72,15 @@ func PointTo[T any](x T) *T {
 func GetApplicationSelector(applicationName string) map[string]string {
 	return map[string]string{"app": applicationName}
 }
+
+func HasUpperCaseLetter(word string) bool {
+	hasUpper := false
+	for _, letter := range word {
+		if unicode.IsUpper(letter) {
+			hasUpper = true
+			break
```

Comment:
```suggestion
			return true
```

Would an early return here work better?